### ISSUE TITLE
[Prometheus integration] Improve user experience

### ISSUE THREAD

**Issue by tetianakravchenko** [elastic/integrations#3687]
As an outcome of the evaluation issues #3487 #3488 #3489:

- https://github.com/elastic/kibana/issues/135981
- https://github.com/elastic/integrations/issues/3668 - ootb dashboard

Another action point for prometheus integration would be to improve user experience (similar to https://github.com/elastic/kibana/issues/134747), but we can come back to it later after k8s integration https://github.com/elastic/kibana/issues/134747 is done to use it as an example.

**Comment by tetianakravchenko** [comment]
TODO:
- align http configuration for datasets

**Comment by pmeresanu85** [comment]
@tetianakravchenko can we consider the Prometheus integration as complete?

**Comment by rameshelastic** [comment]
@pmeresanu85 , while key tasks are complete, we will be carrying over a couple of items (https://github.com/elastic/integrations/issues/3668 etc)  to complete in 8.6

**Comment by pmeresanu85** [comment]
@rameshelastic does this mean that the Prometheus integration is not GA with 8.5?

**Comment by tetianakravchenko** [comment]
@pmeresanu85 yes, prometheus integration is not yet GA, the only missing part - https://github.com/elastic/integrations/issues/3668 as mentioned above

**Comment by pmeresanu85** [comment]
Thank you @tetianakravchenko . @mlunadia please remove prometheus GA from : https://docs.google.com/document/d/1g87dXPv1rcjrZNk9Wga_AeQfUZG5yYl29UWoXfC26Kw/edit# for the 8.5 launch

**Comment by gizas** [comment]
> TODO: need to be promoted

Promotion done today
